aws-operator-role/iam-policy.json

```diff
@@ -160,7 +159,7 @@
             ],
             "Resource": "${arn_prefix}:iam::${account_id}:role/aws-service-role/*"
         },
-	{
+    {
```

Comment:
this was correct :)

Reply:
![image](https://user-images.githubusercontent.com/1133954/234598722-37fffb3b-51ac-4a5b-83bd-c47d35d215bc.png)
I think this is an issue in the PR preview, I can see it fine on my editor and github online editor